grabbit/src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -178,6 +187,23 @@ class JcrNodesProcessor implements ItemProcessor<JcrNode, Node> {
     }
 
     /**
+     * Returns the "jcr:created", "jcr:lastModified" or "cq:lastModified" date property
+     * for current Jcr Node
+     * Returns null of none of the 3 are found
+     * @return
+     */
+    private static Date getModifiedDate(JcrNode jcrNode) {
+        final String CQ_LAST_MODIFIED = "cq:lastModified"
+        if (jcrNode.hasProperty(JCR_LASTMODIFIED)) {
+            return jcrNode.getProperty(JCR_LASTMODIFIED).date.time
+        }
+        else if (jcrNode.hasProperty(CQ_LAST_MODIFIED)) {
+            return jcrNode.getProperty(CQ_LAST_MODIFIED).date.time
+        }
+        return null
+    }
```

Comment:
Interestingly, I tried swapping them the first time I was looking for a solution to this, and it caused grabbit to fail... upon trying again though I must've done something different, as my tests are passing. I'll remove the getModifiedDate method unless I run into issues again.